Tasks/DownloadSecureFileV1/package.json

```diff
@@ -17,8 +17,12 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.0.0",
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
     "azure-pipelines-task-lib": "^2.8.0",
-    "azure-pipelines-tasks-securefiles-common": "1.178.0"
+    "azure-pipelines-tasks-securefiles-common": "1.179.0"
```

Comment:
Why do we change dependencies other than securefiles-common here?

Reply:
Because DownloadSecureFileV1 and InstallSSHKeyV0 tasks is not migrated on node 10 and they are using non preview version of azure-pipelines-tasks-securefiles-common package for now.

Reply:
Ah Wait, sorry seems I misunderstood you at first. We need to update this dependencies to resolve type errors that appeared after updating to the latest version of securefiles-common.

Reply:
Ok, got it. Could these changes possibly cause any issues with node6 execution handler that is present in these tasks right now?

Reply:
I think no, because in fact this dependencies is using on the step of developing to provide intellisense feature and while building of ts code for type analyzing. Node is executing compiled java script code which these changes do not affect in any way.

Reply:
The only concern I have here is about typescript 4.0.2 dev dependency. We usually use tasklib 3.x for this version of typescript

Reply:
Hmm, @egor-bryzgalov could you please help here. Do I really need to update the tasklib package to 3.x version? As I know 3.x version is for migrating to Node 10.

Reply:
@EzzhevNikita I believe you don't think to change task-lib version here since this task targets Node 6. I also think that it's ok to use typescript 4.0.2 if the task is compiled successfully. Have you conducted any sanity testing for these tasks?

cc: @DaniilShmelev

Reply:
Yes sure, I've tested that. It seems that changing of this dependencies isn't causing any issues